app/views/components/_quick-escape-banner.html.erb

```diff
@@ -3,6 +3,12 @@
   aria_label ||= "Hide this page. Quick exit button, it opens BBC Weather in a new tab"
```

Comment:
This label is a bit of a pain configuration wise because it uses app specific information (BBC weather) which isn't part of the component and will need to be kept in sync with whatever is set for `onboarding_information`. Could we make this more generic so that there isn't something that needs to be kept in sync? otherwise it could be very confusing for a user if it's wrong.